packages/graphql-hooks/src/GraphQLClient.ts

```diff
@@ -200,6 +201,12 @@ class GraphQLClient {
     }
   }
 
+  removeCache(cacheKey) {
+    if (this.cache) {
```

Comment:
we can remove the if doing just:
```
this.cache?.delete(cacheKey)
```

Reply:
made in https://github.com/nearform/graphql-hooks/pull/928/commits/4ea3bd85457943db1b1032920ce4d5d03901d171